src/App.tsx

```diff
@@ -1,14 +1,55 @@
 /* eslint-disable max-len */
-import React from 'react';
+import { useEffect, useState, FC } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
+import { getTodos } from './api';
+import { Todo } from './types/Todo';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
 
-export const App: React.FC = () => {
+export const App: FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [preparedTodos, setPreparedTodos] = useState<Todo[]>([]);
+  const [sorted, setSorted] = useState('');
+  const [query, setQuery] = useState('');
+  const [modalTodo, setModalTodo] = useState<Todo | undefined>();
+
+  useEffect(() => {
+    getTodos().then(data => {
+      setTodos(data);
+      setPreparedTodos(data);
+    });
+  }, []);
+
+  useEffect(() => {
+    const preparingTodos = () => {
```

Comment:
and write `const preparedTodos = preparingTodos()`(don't forget to rewrite this function to return the filtered array)